jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/impl/PostgreSQLSqlDialect.java

```diff
@@ -298,6 +303,21 @@ public DataType dialectSpecificMapJdbcType(final JdbcTypeDescription jdbcTypeDes
         return colType;
     }
 
+    @Override
+    public MappedTable mapTable(final ResultSet tables, final List<String> ignoreErrorList) throws SQLException {
+        final String tableName = tables.getString("TABLE_NAME");
+        if (!ignoreErrorList.contains(POSTGRES_IGNORE_UPPERCASE_TABLES) && containsUppercaseCharacter(tableName)) {
+            throw new IllegalArgumentException("Table " + tableName + " cannot be used in virtual schema. " +
```

Comment:
I would explain why, e.g.
cannot be used in a virtual schema because the table name contains upper case characters, which is not supported.

Reply:
We will add an error handbool and introduce error numbers (#77). This will improved the error messages.